addon/components/base/bs-form/element.js

```diff
@@ -330,7 +330,7 @@ const nonDefaultLayouts = A([
  */
 export default FormGroup.extend({
   layout,
-  classNameBindings: ['disabled:is-disabled', 'required:is-required', 'isValidating'],
+  classNameBindings: ['disabled:disabled', 'required:is-required', 'isValidating'],
```

Comment:
These classes were never part of the official ones defined by Bootstrap, so is there a good reason to rename `is-disabled` to `disabled`? That would be a breaking change, which I would like to prevent if possible...